.ladle/adapters/PlainComponentAdapter.tsx

```diff
@@ -0,0 +1,718 @@
+import type { TextInputProps } from '../../src/components/Common/UI/TextInput/TextInputTypes'
+import type { NumberInputProps } from '../../src/components/Common/UI/NumberInput/NumberInputTypes'
+import type { CheckboxGroupProps } from '../../src/components/Common/UI/CheckboxGroup/CheckboxGroupTypes'
+import type { ComboBoxProps } from '../../src/components/Common/UI/ComboBox/ComboBoxTypes'
+import type { CheckboxProps } from '../../src/components/Common/UI/Checkbox/CheckboxTypes'
+import type { DatePickerProps } from '../../src/components/Common/UI/DatePicker/DatePickerTypes'
+import type { RadioProps } from '../../src/components/Common/UI/Radio/RadioTypes'
+import type { RadioGroupProps } from '../../src/components/Common/UI/RadioGroup/RadioGroupTypes'
+import type { SelectProps } from '../../src/components/Common/UI/Select/SelectTypes'
+import type { SwitchProps } from '../../src/components/Common/UI/Switch/SwitchTypes'
+import type { ComponentsContextType } from '../../src/contexts/ComponentAdapter/ComponentsProvider'
+
+export const PlainComponentAdapter: ComponentsContextType = {
+  TextInput: ({
+    label,
+    description,
+    errorMessage,
+    isRequired,
+    isDisabled,
+    isInvalid,
+    id,
+    name,
+    value,
+    placeholder,
+    onChange,
+    onBlur,
+    inputRef,
+    shouldVisuallyHideLabel,
+    ...props
+  }: TextInputProps) => {
+    const inputId = id || `text-input-${name}`
+    const descriptionId = description ? `${inputId}-description` : undefined
+    const errorMessageId = errorMessage ? `${inputId}-error` : undefined
+    const ariaDescribedby = [descriptionId, errorMessageId].filter(Boolean).join(' ') || undefined
+
+    return (
+      <div className="field-layout">
+        {!shouldVisuallyHideLabel && (
+          <label htmlFor={inputId}>
+            {label}
+            {isRequired && <span aria-hidden="true"> *</span>}
+          </label>
+        )}
+        {shouldVisuallyHideLabel && (
+          <label htmlFor={inputId} className="visually-hidden">
```

Comment:
Nit: i'm assuming we need a stylesheet that is ladle specific that would include this?

Reply:
Oh yeah for sure, I think there is going to be a point where we comb the adapter for bugs, this is more POC in our Playground at the moment.